lib/tictactoe/Bot.ex

```diff
@@ -1,18 +1,18 @@
-defmodule Tictactoe.Bot do
-  @callback calculate_move(Map) :: integer
+defmodule Bot do
+  def calculate_move(:PlayerWon, _, _, _, _), do: nil
+  def calculate_move(:Draw, _, _, _, _), do: nil
 
-  def calculate_move(game_round) do
-      player = :PLAYER_1
-      bot = :PLAYER_2
-      board = game_round.board
-      space = Enum.find_index(
-        board, fn s ->
-          s != player &&
-          s != bot end)
+  def calculate_move(:Playing, board, _, _, :BOT_EASY) do
+    Board.available_spaces(board)
```

Comment:
tradition has it to start the pipe with the bare arguments, so
```
board
|> Board.available_spaces()
|> BotIntellect.random_move()
```

Reply:
Thx